autoTatort.py

```diff
@@ -230,6 +230,10 @@ def checkForHDFile(url):
 				mediaURL = stream
 				debug("We have only one stream. Will download it")
 
+			# ensure we always have a valid protocol
+			if not "https:" in mediaURL:
```

Comment:
Maybe this should be `if not mediaURL.startswith("https:")`? Or even better: `if not mediaURL.startswith("http")`? The second one will not accidentally put https: in front of http URLs

Reply:
Thanks for the suggestions. I agree the second one is the better way to go. Updated the PR accordingly.